api/query/cache/service/search.go

```diff
@@ -0,0 +1,220 @@
+// Copyright 2018 The WPT Dashboard Project. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/web-platform-tests/wpt.fyi/api/query"
+	"github.com/web-platform-tests/wpt.fyi/api/query/cache/index"
+	cq "github.com/web-platform-tests/wpt.fyi/api/query/cache/query"
+	"github.com/web-platform-tests/wpt.fyi/shared"
+)
+
+type searchErr struct {
+	// Detail is the internal (inner) error that should not be exposed.
```

Comment:
```suggestion
	// Detail is the internal error that should not be exposed to the end-user.
```

This just seems to be defining the word 'internal'?